war/src/main/scss/modules/page-header.scss

```diff
@@ -1,5 +1,4 @@
 .page-header {
-  display: block;
   display: flex;
```

Comment:
Checking with weekly.ci, `display: flex;` took precedence before and keeps the current layout in place. I assume `block` was added unintentionally, given it's ignored atm @janfaracik ?

Reply:
it was possibly for older browsers that we no longer support